pkg/visor/visorconfig/parse.go

```diff
@@ -28,7 +28,16 @@ func Parse(log *logging.Logger, r io.Reader, confPath string, visorBuildInfo *bu
 
 	// we check if the version of the visor and config are the same
 	if (conf.Version != "unknown") && (visorBuildInfo.Version != "unknown") {
-		compat = strings.Contains(strings.Split(visorBuildInfo.Version, "-")[0], strings.Split(conf.Version, "-")[0])
+		cver := strings.Split(visorBuildInfo.Version, "-")[0] //v0.6.0
```

Comment:
Can we use this package to compare versions? Only need to compare major version I think https://pkg.go.dev/golang.org/x/mod/semver